### PR TITLE
fix login/forget password button overlaps password input

### LINKED ISSUE #15466: Login/forgot password buttons overlap password input when keyboard is active on login page for Devices with small screens
cc @Parveshdhull 

Reproducible on iPhone SE. Can be reproduced on emulated devices with small screens

**Steps:**

1. Open login screen
2. Open keyboard
3. See Login/forgot password buttons location

**Actual result:** Login/forgot password buttons overlap password input

Android Emulator - Nexus_One_API_32:5554 2023-03-24 11-27-24

https://user-images.githubusercontent.com/97245802/227481044-2eb1cd6c-f57e-4150-83b4-406522fda4f3.mp4

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/15466

status: ready

### CODE REVIEW

**Review by ibrkhalil — APPROVED** [review]

**Comment by pavloburykh** [comment]
@Parveshdhull thanx for the PR! Could you please rebase it to the latest develop as there are some important e2e fixes? Thanx.

File: src/status_im2/contexts/onboarding/profiles/view.cljs [hunk 1]
```diff
@@ -135,18 +135,19 @@
...
+       :type                :blur-bg
+       :icon                true
+       :on-press            #(reset! show-profiles? true)
+       :override-theme      :dark
+       :width               32
+       :accessibility-label :show-profiles
+       :style               style/multi-profile-button}
+      :i/multi-profile]
+     [rn/scroll-view
+      {:keyboardShouldPersistTaps :always
```
Comment (J-Son89) [inline src/status_im2/contexts/onboarding/profiles/view.cljs]: you can make kebab case afaik

Comment (Parveshdhull) [reply src/status_im2/contexts/onboarding/profiles/view.cljs]: changed, thank you

**Review by J-Son89 — APPROVED** [review]

**Comment by pavloburykh** [comment]
@Parveshdhull thanx for the PR! Sorry for the delay, we were experiencing problems with e2e runs today, it took quite a long time before we at last got a successful run. PR is ready to be merged.

**Pull request merged**